CONTRIBUTING.md

```diff
@@ -6,21 +6,21 @@
 
 If you think you found a bug in Tesseract, please create an issue.
 
-Use the [users mailing-list](https://groups.google.com/d/forum/tesseract-ocr) instead of creating an Issue if ...
+Use the [users mailing-list](https://groups.google.com/forum/#!forum/tesseract-ocr) instead of creating an Issue if ...
 * You have problems using Tesseract and need some help.
 * You have problems installing the software.
 * You are not satisfied with the accuracy of the OCR, and want to ask how you can improve it. Note: You should first read the [ImproveQuality](https://github.com/tesseract-ocr/tesseract/wiki/ImproveQuality) wiki page. 
-* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki/tesstrain.sh) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
+* You are trying to train Tesseract and you have a problem and/or want to ask a question about the training process.  Note: You should first read the **official** guides [[1]](https://github.com/tesseract-ocr/tesseract/wiki) or [[2]](https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract) found in the project wiki.
```

Comment:
I‌ think the first link should be https://github.com/tesseract-ocr/tesseract/wiki/Training-Tesseract-%E2%80%93-tesstrain.sh. It was renamed with Wiki commit a8d1ec6bcc941c3564666a6b12ab7208285cacbd.